doc(FEC 12554): create browsers support document + add missing player config documentation (#567)
kaltura/kaltura-player-js: docs/playlist.md

For the complete `playlist` API, see [PlaylistManager](./api.md#playlistmanager).

## Playlist Events

> Once a playlist load is requested async calls will be called until the playlist object is ready for use. Inorder to use the api correctly you must be aware that the playlist is ready for use after the playtist is loaded (witing for `PLAYLIST_LOADED` event).

Reviewer: ```suggestion
Once a playlist load is requested, async api calls are made. You can use the playlist api after the playlist data is loaded (`PLAYLIST_LOADED` event).
```

Reviewer 2: The wording is unclear, what's the point?

[Playlist Events](https://github.com/kaltura/kaltura-player-js/blob/master/src/common/playlist/playlist-event-type.js)

 * `PLAYLIST_LOADED`
 * `PLAYLIST_ITEM_CHANGED` (multiple)
 * `PLAYLIST_ENDED`

> ## Listening to Event before calling playlist API the first time

Reviewer: ```suggestion
## Listening to an Event before calling the playlist API the first time
```

Reviewer 2: "for the first time" instead of "at the first time"

  console.log(kalturaPlayer.playlist.options.autoContinue); // true
});
```

## Error Handling

> Error handling ise based on player errors which may result from BE requests or media issues 

Reviewer: ```suggestion
Error handling is based on player errors which may result from Kaltura api requests or media issues 
```
